### PR TITLE
BaSP-TG-69: Task Form

### PR DESCRIPTION
**Pull request by Facund0Fernandez**
- Delete older component.
- Implement inputs shared component.
- Implement buttons.
- Implement styles.
- Implement modal.
- Add redirect to list on form submit
add-form
update-form

### CODE REVIEW

File: src/Components/Task/index.jsx [hunk 1]
```diff
@@ -33,6 +40,7 @@ const Form = () => {
           setDateValue(date);
           setWorkedHoursValue(response.data.workedHours);
           setProjectNameValue(response.data.projectId.name);
+          console.log(projectNameValue);
```
Comment (Gearyandres94) [inline src/Components/Task/index.jsx:43]: Delete this console.log

**Review by Gearyandres94 — CHANGES_REQUESTED** [review]
The select works, but it doesn't show the value I selected. The select should be longer. The modal doesn't have a title and a text.  The button should say "Add new Task" or "Update Task". The error from the database should be a modal.

**Review by Gearyandres94 — APPROVED** [review]
Great Job!!!

**Review by Gearyandres94 — DISMISSED** [review]
Good Job!!!!

**Review by GuidoCerioni — APPROVED** [review]
Nice job!!

**Review by vadalamati — APPROVED** [review]
Goob job!!!!

**Review by Gearyandres94 — APPROVED** [review]

**Review by FiolAxel — APPROVED** [review]

**Review by vadalamati — APPROVED** [review]
Good job!!!

**Pull request merged**